components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -7729,6 +7729,16 @@ paths:
           description: -Not supported yet-
           schema:
             type: string
+        - name: name
```

Comment:
Is there a reason why we need this?

Reply:
Fixed by [b7f664a](https://github.com/wso2/carbon-apimgt/pull/11325/commits/b7f664af4e393bb1fe5d307dd9dc9c9107e372d6)